src/tests/performVerifySorting.spec.js

```diff
@@ -0,0 +1,38 @@
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+const { login, password } = require('../auth/user.json');
+
+test.describe('Inventory Sorting', () => {
+    const testParams = [
+        { name: 'az', sort: [] },
```

Comment:
Let`s improve the test date, for example, add a bit about the scenario name.
Sort items by ${elem.name} - we will have for example Sort items by az or Sort items by lohi - what is it mans 
People can be confused if they see such names of scenarios in the report.
Does it make sense to have empty arrays in the test data?